im2txt/im2txt/inference_utils/caption_generator.py

```diff
@@ -54,6 +54,16 @@ def __cmp__(self, other):
       return -1
     else:
       return 1
+  
+  # for version 3 compatibility (__cmp__ is deprecated)
```

Comment:
Thanks for the contribution!
Please capitalize the first word and add a period in this comment.
Also, please say "Python 3" instead of "version 3".